README.md

```diff
@@ -160,6 +112,34 @@ To perform actions, simply run ```./immuclient``` on the same machine or ```./im
 
 To manage the database, simply run ```./immuadmin``` on the same machine. You can also find immuadmin in the immudb container at `/usr/local/bin/immuadmin` or run a Docker container to connect to the local immudb with `docker run -it --rm --name immuadmin codenotary/immuadmin:latest status`. For security reasons we recommend using immuadmin only on the same system as immudb. User management is restricted to localhost usage.
 
+## Using immudb
```

Comment:
Could go either way on the ordering here; I flipped it because of discussion around wanting potential users to understand how they'd use immudb before giving them quickstart instructions. But I'm not adamant about one over the other.